students/20th/yujinlim/js/main.js

```diff
@@ -0,0 +1,169 @@
+});
+const feedLikeBtns = Array.from(
+  document.querySelectorAll(".feed__feature-btn .fa-heart")
+);
+
+const modal = document.querySelector(".modal-container");
+const MODALOPEN = "modal-open";
+const userBtn = document.querySelector(".top-nav__menu:last-child");
+
+const myID = document.querySelector(".main-right__my-id").innerText;
+const OPACITY = "feed__submit-btn--opacity";
+
+function nthParent(element, n) {
+  while (n-- && element) element = element.parentNode;
+  return element;
+}
+
+function removeSubmitBtnOpacity(commentInputValue, submitBtn) {
+  if (commentInputValue) {
+    submitBtn.classList.remove(OPACITY);
+  }
+
+  if (!commentInputValue) {
+    submitBtn.classList.add(OPACITY);
+  }
+}
+
+function getCommentValues(e) {
+  const commentInputValue = e.target.value;
+  const submitBtn = e.target.parentNode.querySelector(".feed__submit-btn");
+  removeSubmitBtnOpacity(commentInputValue, submitBtn);
+}
+
+function toggleCommentLike() {
+  const isCommentLikePressed = this.querySelector("i").classList.contains(
+    "fas"
+  );
+
+  if (!isCommentLikePressed) {
+    this.querySelector("i").classList.replace("far", "fas");
+  }
+
+  if (isCommentLikePressed) {
+    this.querySelector("i").classList.replace("fas", "far");
+  }
+}
+
+function deleteComment() {
+  const selectedComment = nthParent(this, 2);
+  selectedComment.remove();
+}
+
+function addComment(materialsForAddComment) {
+  materialsForAddComment.commentList.appendChild(
+    materialsForAddComment.commentColumn
+  );
+
+  materialsForAddComment.commentInput.value = "";
+  materialsForAddComment.submitBtn.classList.add(OPACITY);
+
+  const commentDeleteBtns = Array.from(
+    document.getElementsByClassName("delete-btn")
+  );
+  const commentLikeBtns = Array.from(
+    document.getElementsByClassName("like-btn")
+  );
+
+  commentDeleteBtns.forEach((el) => {
+    el.addEventListener("click", deleteComment);
+  });
+  commentLikeBtns.forEach((el) => {
+    el.addEventListener("click", toggleCommentLike);
+  });
+}
+
+function makeComment(e) {
+  const commentColumn = document.createElement("div");
+  const commentContent = document.createElement("span");
+  const commentAuthor = document.createElement("span");
+  const commentBtns = document.createElement("div");
+  commentBtns.classList.add("js-comment-btns");
+
+  const commentInput = e.target.parentNode.querySelector(".feed__input");
+  const commentInputValue = commentInput.value;
+  const commentList = nthParent(e.target, 2)
+    .querySelector(".feed__texts")
+    .querySelector(".js-feed-comments");
+
+  const submitBtn = e.target;
+
+  if (commentInputValue) {
+    commentContent.innerText = commentInputValue;
+    commentAuthor.innerText = myID;
+    commentBtns.innerHTML = `
+    <button class="js-comment-btn like-btn">
+    <i class="far fa-heart"></i>
+    </button>
+    <button class="js-comment-btn delete-btn">
+    <i class="fas fa-times"></i>
+    </button>`;
```

Comment:
```suggestion
    <button class="js-comment-btn like-btn">
      <i class="far fa-heart"></i>
    </button>
    <button class="js-comment-btn delete-btn">
      <i class="fas fa-times"></i>
    </button>`;
```
indent 표현해주면 읽기 더 좋을 것 같습ㄴ디ㅏ

Reply:
확인 후 수정 완료했습니다 감사합니다 :)